plotly/tools.py

```diff
@@ -351,12 +351,17 @@ def embed(file_owner_or_url, file_id=None, width="100%", height=525):
     Keyword arguments:
     file_id (default=None) -- an int or string that can be converted to int
                               if you're using a url, don't fill this in!
+    share_key (default=None) -- a string that enables private plots to be
+                                shared. Note, you must have admin privileges
+                                on the file for this to work
     width (default="100%") -- an int or string corresp. to width of the figure
-    height (default="525") -- same as width but corresp. to the height of the figure
+    height (default="525") -- same as width but corresp. to the height of the
+                              figure
 
     """
     try:
-        s = get_embed(file_owner_or_url, file_id, width, height)
+        s = get_embed(file_owner_or_url, file_id, share_key, width, height)
```

Comment:
:cow2: it's more robust to mimic the call signature of the function, in this case, `share_key=share_key`, etc. If you *don't* do this, someone could just insert a kwarg in the call signature later and break this. In fact, this pr *does* insert the `share_key` kwarg just like that.

In general if the arg is specified as positional (use it as a positional, because the *name* might change) and if it's key-wordy (use it as key-wordy, because the *order* might change)